### PR TITLE
Rewrite style filter to use style tokens.

### PR DESCRIPTION
**Pull request by shans**
Implements #30

### CODE REVIEW

**Comment by shans** [comment]
@dglazkov

File: example.exp [hunk 1]
```diff
@@ -9,3 +9,8 @@ case-1: mobile-$1-[nuked].html
 case-2: mobile-$1-[nuked, nostyle].html
 case-3: mobile-$1-[nuked, compressed, nostyle].html
 case-4: mobile-$1-[nuked, extracted].html
+
+chromium: /usr/local/google/home/shanestephens/src/chromium/src
```
Comment (dglazkov) [inline example.exp]: seems weird to check in your local dirs?

**Comment by dglazkov** [comment]
Awesome!

File: example.exp (hunk 1 above)
Comment (shans) [inline example.exp]: Yup, removed.

**Pull request merged**